book/src/evaluation_quickstart.md

```diff
@@ -44,7 +44,7 @@ docker cp server.toml kanidmd:/data/server.toml
 ### Generate evaluation certificates
 
 ```bash
-docker run --rm -i -t -v kanidmd:/data \
+docker run --rm -i -t -v docker.io/kanidmd:/data \
```

Comment:
You changed the volume reference not the container. This is incorrect...

Reply:
Ah dang, you're right. Brain was apparently tried. There was also a third place in the document that I missed to update.

Amended, please check again.